h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -252,10 +256,32 @@ private void validateInput(AutoMLInput input) {
     }
   }
 
+
   private void validateModelValidation(AutoMLBuildSpec buildSpec) {
     if (buildSpec.input_spec.fold_column != null) {
-      eventLog().warn(Stage.Validation, "Fold column "+buildSpec.input_spec.fold_column+" will be used for cross-validation. nfolds parameter will be ignored.");
+      eventLog().warn(Stage.Validation, "Fold column " + buildSpec.input_spec.fold_column + " will be used for cross-validation. nfolds parameter will be ignored.");
       buildSpec.build_control.nfolds = 0;
+    } else if (buildSpec.build_control.nfolds == -1) {
+      Frame trainingFrame = DKV.getGet(buildSpec.input_spec.training_frame);
+      long nrows = trainingFrame.numRows();
+      long ncols = trainingFrame.numCols() - (buildSpec.getNonPredictors().length +
+              (buildSpec.input_spec.ignored_columns != null ? buildSpec.input_spec.ignored_columns.length : 0));
+
+      double max_runtime = buildSpec.build_control.stopping_criteria.max_runtime_secs();
+      long nthreads = Stream.of(H2O.CLOUD.members())
+              .mapToInt((h2o) -> h2o._heartbeat._nthreads)
+              .sum();
+
+      boolean use_blending = ((ncols * nrows) / (max_runtime * nthreads)) > 2064;
+      Log.warn("R coefficient = " + ((ncols * nrows) / (max_runtime * nthreads)));
```

Comment:
why is it a warning? `info` level at best.
`warn` and `error` levels are the 2 levels that potentially require an action from the user: this action is necessary for errors, and recommended/potential for warnings.

Reply:
It's for debugging only (this is still a draft) - `| grep WARN` gives me just the important logs. I have it on my list to remove that before merging. I will keep this open to have another reminder.